packages/jsii-pacmak/test/expected.jsii-calc/dotnet/Amazon.JSII.Tests.CalculatorPackageId/Amazon/JSII/Tests/CalculatorNamespace/IStableInterfaceProxy.cs

```diff
@@ -4,19 +4,13 @@
 
 namespace Amazon.JSII.Tests.CalculatorNamespace
 {
-    /// <remarks>
-    /// <strong>Stability</strong>: Stable
-    /// </remarks>
```

Comment:
Why is this now omitted? Sounds like a useful thing to put in the docs, no?

Reply:
It actually makes more sense to me to do the opposite: stable unless indicated otherwise. We already did that for Python and it makes sense.

It's also how every other API is documented. Nobody else just slathers on "stable" everywhere, they *might* note "deprected" or "experimental" as it applies.